util/invmsspoly.m

```diff
@@ -0,0 +1,2 @@
+function output = invmsspoly(M)
+    output = M^-1;
```

Comment:
how does this work? From the method name I'm assuming that M is an msspoly matrix. But the inverse of a matrix of polynomials is in general not a matrix of polynomials...